### PR TITLE
Add weather to beaches page

### PR DESCRIPTION
**Pull request by Aisha-Yusuff**
Implement the weather api to show the weather for each beach with css styling

### CODE REVIEW

File: client/src/api/WeatherForecast.js [hunk 1]
```diff
@@ -0,0 +1,87 @@
+import { weatherAPI_KEY } from "../weatherApiKey";
```
Comment (Hafizalimi7) [inline client/src/api/WeatherForecast.js:1]: good work !

File: client/src/components/Weather/Weather.css [hunk 2]
```diff
@@ -0,0 +1,62 @@
+.weather-container {
```
Comment (Hafizalimi7) [inline client/src/components/Weather/Weather.css:1]: good css

File: client/src/components/Weather/Weather.jsx [hunk 3]
```diff
@@ -0,0 +1,35 @@
+import { useEffect } from "react";
```
Comment (Hafizalimi7) [inline client/src/components/Weather/Weather.jsx:1]: well written code

**Review by Hafizalimi7 — APPROVED** [review]

**Review by robimaliqi — APPROVED** [review]
great work

**Pull request merged**